docs/connectors/taps/mysql.rst

```diff
@@ -141,6 +141,11 @@ Example YAML for ``tap-mysql``:
   stream_buffer_size: 0                  # In-memory buffer size (MB) between taps and targets for asynchronous data pipes
   #batch_wait_limit_seconds: 3600        # Optional: Maximum time to wait for `batch_size_rows`. Available only for snowflake target.
 
+  # Options only for Fastsycn for Snowflake
```

Comment:
Typo in "Fastsycn". 

Also, mongo & tap-s3-csv do have FastSync with Snowflake

Reply:
typo fixed by https://github.com/transferwise/pipelinewise/pull/696/commits/7680704de2e3a2c21df018847401726e60dda9cd

mongo and s3 are not in the scope of this PR.

S3 and Mongo csv exports both have headers in the export files which causing troubles when splitting files in multiple pieces. `skip_csv_header` should skip the first line only in the first file part or ideally should not be included in the export.

Splitting files is an option only for mysql and postgres exports by this PR.